### PR TITLE
wrap and export template app before importing

### PR DESCRIPTION
**Pull request by snopoke**
http://manage.dimagi.com/default.asp?179392

@benrudolph 

cc @TylerSheffels 
FYI @orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/views.py [hunk 1]
```diff
@@ -348,7 +348,8 @@ def copy_app(request, domain):
 def app_from_template(request, domain, slug):
     _clear_app_cache(request, domain)
     template = load_app_template(slug)
-    app = import_app_util(template, domain, {
+    source = template.export_json(dump_json=False)
```
Comment (snopoke) [inline corehq/apps/app_manager/views.py]: This is the real fix. This function updates all the unique ID's in the app so that we don't get duplicates.

Comment (snopoke) [inline corehq/apps/app_manager/views.py]: I wonder if this shouldn't actually happen in the import_app function to be even more defensive?

**Comment by snopoke** [comment]
Updated do to the wrap and export inside `import_app`

File: corehq/apps/app_manager/views.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/app_manager/views.py]: +1

**Pull request merged**